search/serializers_test.py

```diff
@@ -133,8 +132,7 @@ def test_serialize_bulk_post_and_comments(mocker, patched_base_post_serializer,
         replies=[inner_comment_mock]
     )
     post_mock = mocker.MagicMock(comments=[outer_comment_mock])
-    assert list(serialize_bulk_post_and_comments(post_mock)) == [
-        serialize_post_for_bulk(post_mock),
+    assert list(serialize_bulk_comments(post_mock)) == [
```

Comment:
Can you change the name of this test? Do we have a test for serializing posts?

Reply:
✅ , `test_serialize_post_for_bulk`